pkg/controller/node/status.go

```diff
@@ -118,6 +118,10 @@ func calculateStatus(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) mcfgv
 
 // isNodeManaged checks whether the MCD has ever run on a node
 func isNodeManaged(node *corev1.Node) bool {
+	if isWindows(node) {
```

Comment:
Ideally we might not need this as [this](https://github.com/openshift/machine-config-operator/pull/1823/files#diff-3bd49282ca65e5525da44a27caaa55c7L124) should fail but I wanted to be explicit and fail early.